tests/network/port_forward.go

```diff
@@ -52,17 +52,24 @@ var _ = SIGDescribe("Port-forward", func() {
 		tests.BeforeTestCleanup()
 	})
 
+	//This test is structured as the following:
```

Comment:
There is actually a bug-proof solution. See what we do for namespaces:
```
// CalculateNamespaces checks on which ginkgo gest node the tests are run and sets the namespaces accordingly
func CalculateNamespaces() {
	worker := config.GinkgoConfig.ParallelNode
	util2.NamespaceTestDefault = fmt.Sprintf("%s%d", util2.NamespaceTestDefault, worker)
	NamespaceTestAlternative = fmt.Sprintf("%s%d", NamespaceTestAlternative, worker)
	// TODO, that is not needed, just a shortcut to not have to treat this namespace
	// differently when running in parallel
	NamespaceTestOperator = fmt.Sprintf("%s%d", NamespaceTestOperator, worker)
	TestNamespaces = []string{util2.NamespaceTestDefault, NamespaceTestAlternative, NamespaceTestOperator}
}
```

Similiary we can get "random" port. (150%d, worker)

Reply:
Great. I have just a question about it: what happens if we have 6 tests and 3 nodes? It will runs each couple of them in parallel in the same node or it will runs 3 in parallel and then the other 3? Because in the scenario 1 it will fails the same. Thanks

Reply:
The number of nodes/workers also defines how many tests can run in parallel. Therefore the latter and you are safe.

Reply:
Awesome. Thanks @xpivarc for the explanation